charts/rd-location-ref-data-load/values.yaml

```diff
@@ -0,0 +1,11 @@
+java:
+  applicationPort: 8099
+  image: 'hmctspublic.azurecr.io/rd/location-ref-data-load:latest'
+  ingressHost: rd-location-ref-data-load-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+  aadIdentityName: rd
+  environment:
+  keyVaults:
+    rd:
+      secrets:
+        - AppInsightsInstrumentationKey
+        - rd-location-ref-data-load-s2s-secret
```

Comment:
we don't need  rd-location-ref-data-load-s2s-secret as it is scheduling application.Please refer https://github.com/hmcts/rd-judicial-data-load